src/createR.js

```diff
@@ -4,40 +4,87 @@ const initialState = {
   status: STATUS.IDLE,
   loading: false,
   error: null,
-  data: null
+  data: null,
 }
 
-const createR = name => (state = initialState, action) => {
-  const { type, payload, meta } = action
+const defaultReducer = (state, action) => {
+  const { type, payload } = action
 
-  if (!meta || (meta && meta.name !== name)) return state
-  if (!type.startsWith(prefixAT(name))) return state
+  switch (type) {
+    case STATUS.PENDING:
+      return {
+        error: null,
+        data: null,
+      }
+    case STATUS.FULFILLED:
+      return {
+        error: null,
+        data: payload,
+      }
+    case STATUS.REJECTED:
+      return {
+        error: payload,
+        data: null,
+      }
+    default:
+      return state
+  }
+}
 
-  switch (meta.status) {
+const trackingReducer = (state, action) => {
+  const { type } = action
+
+  switch (type) {
     case STATUS.PENDING:
       return {
         status: STATUS.PENDING,
         loading: true,
-        error: null,
-        data: null
       }
     case STATUS.FULFILLED:
       return {
         status: STATUS.FULFILLED,
         loading: false,
-        error: null,
-        data: payload
       }
     case STATUS.REJECTED:
       return {
         status: STATUS.REJECTED,
         loading: false,
-        error: payload,
-        data: null
       }
     default:
       return state
   }
 }
 
+const guard = name => (action) => {
```

Comment:
Could not use unstructuring of these elements as function parameters?
`cosnt defaultReducer  = ({ type, meta }) => ...`